### PR TITLE
Tune some limits

### PR DESCRIPTION
**Pull request by roman-khimov**
Follow recent Neo changes.

### CODE REVIEW

**Review by AnnaShaleva — APPROVED** [review]

File: pkg/core/transaction/oracle.go [hunk 1]
```diff
@@ -16,7 +17,7 @@ type OracleResponse struct {
 	Result []byte             `json:"result"`
 }
 
-const maxResultSize = 1024
+const MaxOracleResultSize = math.MaxUint16
```
Comment (AnnaShaleva) [inline pkg/core/transaction/oracle.go:20]: lint is unhappy about the comment.

File: pkg/core/blockchain.go [hunk 2]
```diff
@@ -148,6 +149,10 @@ func NewBlockchain(s storage.Store, cfg config.ProtocolConfiguration, log *zap.L
 		cfg.MemPoolSize = defaultMemPoolSize
 		log.Info("mempool size is not set or wrong, setting default value", zap.Int("MemPoolSize", cfg.MemPoolSize))
 	}
+	if cfg.MaxTraceableBlocks == 0 {
```
Comment (fyrchik) [inline pkg/core/blockchain.go:152]: `<= 0` ?

**Review by fyrchik — APPROVED** [review]

File: pkg/core/blockchain.go (hunk 2 above)
Comment (roman-khimov) [reply pkg/core/blockchain.go:152]: It's `uint32`.

File: pkg/core/transaction/oracle.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/core/transaction/oracle.go:21]: Fixed.

**Pull request merged**